### ISSUE TITLE
Ubuntu 16.04: sensu-client init script does not start/stop the daemon

### ISSUE THREAD

**Issue by heffergm** [sensu/sensu#1421]
sensu-client init script fails to start or stop the daemon
## Expected Behavior

Running `sensu-client {start|stop}` should result in the service starting/stopping. Works as expected on Ubuntu 14.04.
## Current Behavior

Running `service sensu-client {start|stop}` results in an immediate return with no output and a 0 exit status. The state of the daemon remains unchanged.

Running `service sensu-service client {start|stop}` works as expected however, and successfully starts/stop the daemon.

The issue here (for me) is that on host shutdown, clients are failing to de-register themselves because the daemon isn't being stopped.
## Steps to Reproduce (for bugs)
1. install sensu 0.25.3-1 from debian package
2. `service sensu-client {start|stop}` will fail to do either
## Context

Clients fail to de-register themselves on node shutdown, since the daemon is not being stopped via init.
## Your Environment
- Sensu version used: 0.25.3-1
- Operating System and version (e.g. Ubuntu 14.04): Ubuntu 16.04 
- Transport and runtime versions (e.g. RabbitMQ 3.6.2 on Erlang 18.2): redis

**Comment by runningman84** [comment]
I hope this pull request would solve that using chef: https://github.com/sensu/sensu-chef/pull/485
Furthermore the corresponding changes should be included in the debian package too.

**Comment by heffergm** [comment]
I'm fine with adding systemd support, but that aside, sysv init should continue to work...

**Comment by heffergm** [comment]
This appears to have been resolved after deploying with the latest (3.0.0) cookbook. Closing...

**Comment by cwjohnston** [comment]
Hi @heffergm, thanks for opening this issue. What I've observed is that Ubuntu 16.04 invokes systemd-sysv-generator to use Sensu's existing init scripts under systemd, and this doesn't work out very well for a few different reasons.

I'm not sure what I've encountered is the same situation that you are describing here, but we are working to improve the situation for systemd and other service management systems.

In hopes of aleviating these issues and providing feature parity (e.g. configuration validation prior to restart) across multiple platforms, we're currently working to [overhaul how we start Sensu](https://github.com/sensu/sensu-build/issues/174) and we anticipate we'll ship these improvements in the 0.27 release.
